releases/note-source/v15_0.rst

```diff
@@ -0,0 +1,239 @@
+.. _release-v15-0:
+
+Spring 2018 Release (v15_0)
+===========================
+
+.. ATTENTION::
+   This will be the last major release that supports Python2. From release v16.0 (Fall 2018) on, only Python3 be explicitly supported. Minor releases of v15 will continue to support Python2.
+
++-------------------------------------------+------------+
+| Source                                    | Identifier |
++===========================================+============+
+| Git tag                                   | 15.0       |
++-------------------------------------------+------------+
+| :doc:`EUPS distrib </install/newinstall>` | v15\_0     |
++-------------------------------------------+------------+
+
+- :ref:`release-v15-0-major-changes`
+- :ref:`release-v15-0-bug-fixes`
+- :ref:`release-v15-0-internal-improvements`
+
+.. - :ref:`release-v15-0-sui`
+.. - :ref:`Qserv and Data Access <release-v15-0-qserv-dax>`
```

Comment:
Are we including any SUI or DAX notes?

Reply:
No because this has always been a "Science Pipelines" release.

If that's not what Fritz and Xiu-Qin expect, we should talk to them.

Reply:
Isn't it an lsst_distrib release? I'm having trouble with the idea that e.g. cat is part of the Science Pipelines.

Regardless, I don't think it matters, except that we did include SUIT and DAX notes in the v13 release (https://pipelines.lsst.io/releases/notes.html#release-v13-0) and I wasn't aware of a decision to do things differently this time. That said, I note that they weren't included in the v14 release, so maybe I'm six months late to this party.

Reply:
Yes, it is `lsst_distrib` and at the moment `cat` is no longer part of it (`datarel` brought it in). I think at some point we decided that Qserv/DAX and SUIT are not at all in sync with any of this so we stopped putting them in the numbered releases (Firefly and Qserv don't get tagged this way).

Reply:
```
EUPS distribution v15_0_rc3 version list. Version 1.0
#product             flavor     version
#--------------------------------------
[...]
cat                  generic    14.0-1-g9341377+2
```

Reply:
Yes. But 15.0 was tagged ages ago whereas we removed mariadb and cat from lsst_distrib 2 weeks ago.

Reply:
So.... cat is part of this release, right? I guess I'm confused about your point.

Reply:
When I said "is no longer part of" I meant "at this point in time". I was trying to say that cat is indeed part of lsst_distrib for v15 but is not part of lsst_distrib now.

Reply:
OK, and FWIW, I did change to Python 2 and Python 3.  I was confused about convention in the sense that I thought there was one when there wasn't.